sdv/multi_table/base.py

```diff
@@ -305,6 +310,8 @@ def fit_processed_data(self, processed_data):
         """
         self._fit(processed_data)
         self._fitted = True
+        self._fitted_date = datetime.datetime.today().strftime('%Y-%m-%d')
```

Comment:
Just minor question, should we log the hour, minute and seconds? I think it would be nice to have it when working with multiple models that got fitted the same day @npatki

Reply:
I would say for now, the `day` might be good enough, as this information can tell us more about what state the SDV ecosystem was in (in terms of releases, update etc.)

If people ask for more info, we can later get more granular in the future.

Reply:
Thank you !